doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,130 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Using Design Parameters
+
+You can read and update parameters that are part of the design.
+In this example, we have a simple design with two associated parameters.
+
++++
+
+## Perform required imports
+
+```{code-cell} ipython3
+import os
+from ansys.geometry.core.modeler import *
+from ansys.geometry.core.parameters import *
+```
+
+The file for this example is in the integration tests folder, so let's get its path.
+
+```{code-cell} ipython3
+current_path = os.getcwd()
+file_path = os.path.join(current_path, os.pardir,  os.pardir, os.pardir, os.pardir, "tests/integration/files/blockwithparameters.dsco")
+# Normalize the path to remove any redundant separators
+file_path = os.path.normpath(file_path)
+```
+
+## Import a design with parameters
+
++++
+
+Importing the model using open_file method of the modeler.
+
+```{code-cell} ipython3
+modeler = Modeler(host="localhost", port= 50051)
```

Comment:
All examples should use the ``launch_modeler`` function when committed. I understand you want to use it for local dev but please adapt it before committing.

Reply:
Yeah I think this was updated some days ago when I realized it wasn't been done properly. On "main" it is already fixed. https://github.com/ansys/pyansys-geometry/blob/b70e431b131ac4c0d9d685412172a55077657e16/doc/source/examples/03_modeling/service_colors.mystnb#L46-L49

Good catch!